packages/laconia-config/src/SecretsManagerConfigConverter.js

```diff
@@ -0,0 +1,50 @@
+module.exports = class SecretsManagerConfigConverter {
+  constructor(secretsManager) {
+    this.secretsManager = secretsManager;
+  }
+
+  async _getParameterMap(secretIds) {
+    const datas = await Promise.all(
+      secretIds.map(secretId =>
+        this.secretsManager
+          .getSecretValue({
+            SecretId: secretId
+          })
+          .promise()
+          .catch(err => {
+            if (err.code === "ResourceNotFoundException") {
```

Comment:
What's the reason of hiding this exception? I feel that if I'm the developer, I would like to know if I have misconfigured / mistyped the secret id.

Reply:
I can't remember what I was thinking... I agree with you and am I going to remove it.

Reply:
I guess if we're not sure, I would suggest for it to fail by default even in the case of multiple secrets. Perhaps we can introduce a way to suppress it in the future if someone came along with a valid use case?

Reply:
Yeah I agree with you. I don't have any use cases in mind. It's easy enough to address it if someone comes up with a use case.

Reply:
Great, I'll release this first thing in the morning tomorrow!